# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/linux24/99-mkdep-arglist-too-long.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

Depending on the working directory where the kernel is build, the file
list can grow to produce an "argument to long" error on mkdep line.
This patch only slits the process to have shorter lines.

	- Alejandro

--- ./Makefile.orig	2003-09-23 22:19:55.000000000 -0400
+++ ./Makefile	2003-09-23 22:28:29.000000000 -0400
@@ -518,7 +518,9 @@
 ifdef CONFIG_MODVERSIONS
 	$(MAKE) update-modverfile
 endif
-	scripts/mkdep -- `find $(FINDHPATH) \( -name SCCS -o -name .svn \) -prune -o -follow -name \*.h ! -name modversions.h -print` > .hdepend
+	for x in $(FINDHPATH); do \
+		scripts/mkdep -- `find $$x \( -name SCCS -o -name .svn \) -prune -o -follow -name \*.h ! -name modversions.h -print` >> .hdepend ; \
+	done
 	scripts/mkdep -- init/*.c > .depend
 
 ifdef CONFIG_MODVERSIONS
